dcc-portal-ui/INTERNATIONALIZATION.md

```diff
@@ -139,5 +139,5 @@ language code. This code can be found in the `.po` file header.
 There are different types of services available to translate the strings. Some of them
 are listed on [Wikipedia](http://en.wikipedia.org/wiki/Gettext#See_also). The `.pot` file 
 generated by Grunt plugin is a standard file format for any translator. This file can be
-found in [translations](dcc-portal-ui/translations/) folder. Use any of the listed services or a translator 
+found in [translations](/translations/) folder. Use any of the listed services or a translator 
```

Comment:
The link (http://en.wikipedia.org/wiki/Gettext#See_also) to Wikipedia I mentioned has the services and IDE I was using.